docs/os/windows/windows-vm.md

```diff
@@ -1,5 +1,4 @@
-:warning: *Some of the information below is outdated ; please read [this forum wiki post](https://forum.qubes-os.org/t/wip-windows-qwt-user-reports/9687) for up to date instructions and users feedback.
-The content below will eventually be updated to reflect the newer 4.1 release and QWT recent development (contributors welcome!)*
+:warning: *The content below has been updated to reflect the newer 4.1 release and QWT recent development - see* [Installing and Using Windows-based VMs in Qubes R4.1](https://github.com/Qubes-Community/Contents/blob/master/docs/os/windows/windows-vm41.md)
```

Comment:
ditto

Reply:
You`'re right: it's somewhat confusing. I changed the text to make it clearer.